hledger/test/incomestatement.test

```diff
@@ -296,6 +296,7 @@ Income Statement ..
 =================++====
  Revenues        ||    
 -----------------++----
+ revenues        ||  0 
```

Comment:
Should drop "leaf" from the test description now I guess.

Reply:
Done